### DISCUSSION TITLE
Frigate detection rate

### DISCUSSION THREAD

**Discussion by n0zo** [blakeblackshear/frigate#9280] [Ask A Question]
I have tried a search and didn't have any luck; I'm guessing I'm using the wrong key words.
Is there a way to set the detection rate, especially for people? 
I'm passing it on to Doubletake; but at times it's getting so much information that it causes CodeProject.AI (Deepstack) to crash to the point where it has to be restarted; and with CompreFace it'll overwhelm it to where sometimes it takes 15 minutes before it starts processing anything.

This is mainly in highly occupied areas, with several people sometimes, and then over several cameras. It can be trying to process as many as an image every second from various cameras, and that's where it seems to hang up. Example being a living room while watching TV, or a kitchen while cooking. 

What I'd like to do is make it so that instead of what looks like a camera processing an image from Frigate to DoubleTake every 10 seconds (per camera); I'd like to bump that up to maybe 30 or 45 seconds for the facial recognition.

That would be a per camera setting option preferably, indoors. Outdoors I would like to leave it as is with the rate it publishes stuff that Doubletake picks up.

**Comment by NickM-27** [comment]
Doubletake has a detection delay between calls that you can configure
